string file_content
---
string result